src/utils/index.js

```diff
@@ -1,11 +1,11 @@
-const emptyEndpointResponse = () => {
-  return { status: "API is running on /api" };
-};
 export function validateBody(reqBody, expected) {
```

Comment:
Please rename the file from index.js to something more descriptive. 
Also the function needs to validate if emails are correct as well.

Reply:
I renamed index.js to validators.js